jbdevstudio/multiple/jbdevstudio-multiple.target

```diff
@@ -456,6 +456,13 @@
       <repository location="http://download.jboss.org/jbosstools/updates/requirements/testng/6.8.6.20130914_0724/"/>
       <unit id="org.testng.eclipse.feature.group" version="6.8.6.20130914_0724"/>
     </location>
+    
+    <!-- Eclipse Docker Tooling -->
+    <location includeAllPlatforms="true" includeMode="slicer" type="InstallableUnit" includeSource="true">
+      <repository location="http://download.jboss.org/jbosstools/updates/requirements/docker/"/>
```

Comment:
We need the version suffix in mirror URL here: http://download.jboss.org/jbosstools/updates/requirements/docker/1.0.0.201506030402/